lib/r10k/environment/name.rb

```diff
@@ -7,7 +7,7 @@ class Name
 
       # @!attribute [r] name
       #   @return [String] The unmodified name of the environment
-      attr_reader :name
+      attr_reader :name, :ref
```

Comment:
Possibly a nit: this new attribute, `:ref`, is not documented. I think it needs to be. Without understanding exactly what it is for, and what the difference between it and name is, it would probably be a little bit confusing to look at later and try to figure out what was going on and why.

Reply:
have added something now